crates/http-sink/hub/package-meta.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 package_format_version: "0.3"
-name: http
```

Comment:
Sure!

Reply:
Actually @sehyo, I think this is required, because the `ipkg` name is generated from this field `name` in package-meta file.

Checking on the Source HTTP Connector, we can see that the same approach is done:

https://github.com/infinyon/http-source-connector/blob/main/crates/http-source/hub/package-meta.yaml

Reply:
thats incorrect.  try dry run to see if it does

Reply:
Great @sehz! It worked with `cdk publish --pack -p http-sink`!